package.json

```diff
@@ -11,7 +11,18 @@
   "configSchema": {
     "executablePath": {
       "type": "string",
-      "default": "codeclimate"
+      "default": "codeclimate",
+      "description": "The path to the `codeclimate` executable."
```

Comment:
The problem is that users quite often will stick a directory in there since using "path" to define the complete path of something is really only done by programmers from what I've seen.

Anyway, full or absolute works here 😉.